testing-and-tuning/sample_conf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "date_format": "%m/%d/%Y",
+  "initial_id": 100000,
+  "translation_map": {
+    "given_name": "First Name",
+    "family_name": "Last Name",
+    "DOB": "DateOfBirth",
+    "phone": "Telephone Number",
+    "address": "Street Address 1",
+    "zip": "Zip",
+    "default_values": {
+      "sex": "F"
+    }
+  },
+  "regularization_rules": {
```

Comment:
One last thought as you're addressing other comments, the term "regularization" isn't something I've ever heard before. From a quick google search it seems to primarily be used in machine learning. I get what you're going for but could we call it "code mapping" or "value mapping" or something like that?